2022-06-09 01:11:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "0 (CLIENTE: ""DATANASCITA"" DATE)"; SQL statement:
INSERT INTO CLIENTE VALUES(1, 'Kurt', 'Cobain', 20/02/1967, 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', 171, 98112, 'A1') [22018-212]
2022-06-09 01:11:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "0 (CLIENTE: ""DATANASCITA"" DATE)"; SQL statement:
INSERT INTO CLIENTE VALUES(2, 'Dave', 'Grohl', 14/01/1969, 'dave.grohl69@gmail.com', 'Gable Dr Encino', 172, 91316, 'A2') [22018-212]
2022-06-09 01:11:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "0 (CLIENTE: ""DATANASCITA"" DATE)"; SQL statement:
INSERT INTO CLIENTE VALUES(3, 'Krist', 'Novoselic', 16/05/1965, 'krist.novo65@gmail.com', 'S Lucile St', 173, 98148, 'AM') [22018-212]
2022-06-09 01:13:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "0 (CLIENTE: ""DATANASCITA"" DATE)"; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', 20/02/1967, 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', 171, 98112, 'A1') [22018-212]
2022-06-09 01:13:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "0 (CLIENTE: ""DATANASCITA"" DATE)"; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', 20/02/1967, 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', 171, 98112, 'A1') [22018-212]
2022-06-09 01:13:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "20/02/1967"; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', '20/02/1967', 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', 171, 98112, 'A1') [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.value.Value.convertTo(Value.java:1147)
	at org.h2.value.Value.convertForAssignTo(Value.java:1092)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:369)
	at org.h2.table.Table.convertInsertRow(Table.java:931)
	at org.h2.command.dml.Insert.insertRows(Insert.java:167)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NumberFormatException: For input string: "20/02/"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:668)
	at java.base/java.lang.Integer.parseInt(Integer.java:784)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:190)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 19 more
2022-06-09 01:13:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "-1949 (CLIENTE: ""DATANASCITA"" DATE)"; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', 20-02-1967, 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', 171, 98112, 'A1') [22018-212]
2022-06-09 01:14:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "0 (CLIENTE: ""DATANASCITA"" DATE)"; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', 20/02/1967, 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', 171, 98112, 'A1') [22018-212]
2022-06-09 01:15:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "20/02/1967"; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', '20/02/1967', 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', 171, 98112, 'A1') [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.value.Value.convertTo(Value.java:1147)
	at org.h2.value.Value.convertForAssignTo(Value.java:1092)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:369)
	at org.h2.table.Table.convertInsertRow(Table.java:931)
	at org.h2.command.dml.Insert.insertRows(Insert.java:167)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NumberFormatException: For input string: "20/02/"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:668)
	at java.base/java.lang.Integer.parseInt(Integer.java:784)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:190)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 19 more
2022-06-09 01:15:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "20/02/1967"; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', '20/02/1967', 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', '171', '98112', 'A1') [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.value.Value.convertTo(Value.java:1147)
	at org.h2.value.Value.convertForAssignTo(Value.java:1092)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:369)
	at org.h2.table.Table.convertInsertRow(Table.java:931)
	at org.h2.command.dml.Insert.insertRows(Insert.java:167)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NumberFormatException: For input string: "20/02/"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:668)
	at java.base/java.lang.Integer.parseInt(Integer.java:784)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:190)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 19 more
2022-06-09 01:15:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "20/02/1967"; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', '20/02/1967', 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', '171', '98112', 'A1') [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.value.Value.convertTo(Value.java:1147)
	at org.h2.value.Value.convertForAssignTo(Value.java:1092)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:369)
	at org.h2.table.Table.convertInsertRow(Table.java:931)
	at org.h2.command.dml.Insert.insertRows(Insert.java:167)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NumberFormatException: For input string: "20/02/"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:668)
	at java.base/java.lang.Integer.parseInt(Integer.java:784)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:190)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 19 more
2022-06-09 01:15:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "20/02/1967"; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', '20/02/1967', 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', '171', '98112', 'A1') [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.value.Value.convertTo(Value.java:1147)
	at org.h2.value.Value.convertForAssignTo(Value.java:1092)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:369)
	at org.h2.table.Table.convertInsertRow(Table.java:931)
	at org.h2.command.dml.Insert.insertRows(Insert.java:167)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NumberFormatException: For input string: "20/02/"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:668)
	at java.base/java.lang.Integer.parseInt(Integer.java:784)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:190)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 19 more
2022-06-09 01:15:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "20/02/1967"; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', '20/02/1967', 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', '171', '98112', 'A1') [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.value.Value.convertTo(Value.java:1147)
	at org.h2.value.Value.convertForAssignTo(Value.java:1092)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:369)
	at org.h2.table.Table.convertInsertRow(Table.java:931)
	at org.h2.command.dml.Insert.insertRows(Insert.java:167)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NumberFormatException: For input string: "20/02/"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:668)
	at java.base/java.lang.Integer.parseInt(Integer.java:784)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:190)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 19 more
2022-06-09 01:15:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "0 (CLIENTE: ""DATANASCITA"" DATE)"; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', 20/02/1967, 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', '171', '98112', 'A1') [22018-212]
2022-06-09 01:16:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Function "TO_DATE" not found; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', TO_DATE('20/02/1967'), 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', '171', '98112', 'A1') [90022-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.readFunction(Parser.java:3894)
	at org.h2.command.Parser.readTerm(Parser.java:5227)
	at org.h2.command.Parser.readFactor(Parser.java:3440)
	at org.h2.command.Parser.readSum(Parser.java:3427)
	at org.h2.command.Parser.readConcat(Parser.java:3392)
	at org.h2.command.Parser.readCondition(Parser.java:3177)
	at org.h2.command.Parser.readExpression(Parser.java:3093)
	at org.h2.command.Parser.readExpressionOrDefault(Parser.java:3079)
	at org.h2.command.Parser.parseValuesForCommand(Parser.java:1800)
	at org.h2.command.Parser.parseInsert(Parser.java:1671)
	at org.h2.command.Parser.parsePrepared(Parser.java:814)
	at org.h2.command.Parser.parse(Parser.java:691)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:568)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-09 01:18:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "1945 (CLIENTE: ""DATANASCITA"" DATE)"; SQL statement:
INSERT INTO CLIENTE VALUES('AB345678', 'Kurt', 'Cobain', 1967-02-20, 'kurt.cobain67@gmail.com', 'Lake Washington Blvd', '171', '98112', 'A1') [22018-212]
2022-06-09 01:22:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "NUMEROCARTA CHARACTER(8)": "U&'CD91234\\00a0\\00a0\\00a0' (10)"; SQL statement:
UPDATE "PUBLIC"."CLIENTE" SET "NUMEROCARTA"=? ,"NOME"=? ,"COGNOME"=? ,"DATANASCITA"=? ,"EMAIL"=? ,"VIA"=? ,"CIVICO"=? ,"CAP"=? ,"PATENTEDACONSEGUIRE"=?  WHERE "NUMEROCARTA"=? [22001-212]
2022-06-09 01:22:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "NUMEROCARTA CHARACTER(8)": "U&'CD91234\\00a0\\00a0\\00a0' (10)"; SQL statement:
UPDATE "PUBLIC"."CLIENTE" SET "NUMEROCARTA"=? ,"NOME"=? ,"COGNOME"=? ,"DATANASCITA"=? ,"EMAIL"=? ,"VIA"=? ,"CIVICO"=? ,"CAP"=? ,"PATENTEDACONSEGUIRE"=?  WHERE "NUMEROCARTA"=? [22001-212]
2022-06-09 01:22:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "NUMEROCARTA CHARACTER(8)": "U&'EF12345\\00a0\\00a0\\00a0\\00a0\\00a0' (12)"; SQL statement:
UPDATE "PUBLIC"."CLIENTE" SET "NUMEROCARTA"=? ,"NOME"=? ,"COGNOME"=? ,"DATANASCITA"=? ,"EMAIL"=? ,"VIA"=? ,"CIVICO"=? ,"CAP"=? ,"PATENTEDACONSEGUIRE"=?  WHERE "NUMEROCARTA"=? [22001-212]
2022-06-09 01:22:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "NUMEROCARTA CHARACTER(8)": "U&'EF12345\\00a0\\00a0\\00a0\\00a0\\00a0' (12)"; SQL statement:
UPDATE "PUBLIC"."CLIENTE" SET "NUMEROCARTA"=? ,"NOME"=? ,"COGNOME"=? ,"DATANASCITA"=? ,"EMAIL"=? ,"VIA"=? ,"CIVICO"=? ,"CAP"=? ,"PATENTEDACONSEGUIRE"=?  WHERE "NUMEROCARTA"=? [22001-212]
2022-06-09 01:23:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "NUMEROCARTA CHARACTER(8)": "U&'EF678912\\00a0\\00a0\\00a0' (11)"; SQL statement:
UPDATE "PUBLIC"."CLIENTE" SET "NUMEROCARTA"=? ,"NOME"=? ,"COGNOME"=? ,"DATANASCITA"=? ,"EMAIL"=? ,"VIA"=? ,"CIVICO"=? ,"CAP"=? ,"PATENTEDACONSEGUIRE"=?  WHERE "NUMEROCARTA"=? [22001-212]
2022-06-09 01:23:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "NUMEROCARTA CHARACTER(8)": "U&'EF678912\\00a0\\00a0\\00a0' (11)"; SQL statement:
UPDATE "PUBLIC"."CLIENTE" SET "NUMEROCARTA"=? ,"NOME"=? ,"COGNOME"=? ,"DATANASCITA"=? ,"EMAIL"=? ,"VIA"=? ,"CIVICO"=? ,"CAP"=? ,"PATENTEDACONSEGUIRE"=?  WHERE "NUMEROCARTA"=? [22001-212]
2022-06-09 01:23:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "NUMEROCARTA CHARACTER(8)": "U&'EF67891\\00a0\\00a0\\00a0\\00a0\\00a0' (12)"; SQL statement:
UPDATE "PUBLIC"."CLIENTE" SET "NUMEROCARTA"=? ,"NOME"=? ,"COGNOME"=? ,"DATANASCITA"=? ,"EMAIL"=? ,"VIA"=? ,"CIVICO"=? ,"CAP"=? ,"PATENTEDACONSEGUIRE"=?  WHERE "NUMEROCARTA"=? [22001-212]
2022-06-09 01:23:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "NUMEROCARTA CHARACTER(8)": "U&'EF67891\\00a0\\00a0\\00a0\\00a0\\00a0' (12)"; SQL statement:
UPDATE "PUBLIC"."CLIENTE" SET "NUMEROCARTA"=? ,"NOME"=? ,"COGNOME"=? ,"DATANASCITA"=? ,"EMAIL"=? ,"VIA"=? ,"CIVICO"=? ,"CAP"=? ,"PATENTEDACONSEGUIRE"=?  WHERE "NUMEROCARTA"=? [22001-212]
2022-06-09 17:12:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.NullPointerException: Cannot invoke ""String.length()"" because ""<parameter1>"" is null" [50000-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.message.DbException.toSQLException(DbException.java:367)
	at org.h2.message.TraceObject.logAndConvert(TraceObject.java:365)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:225)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NullPointerException: Cannot invoke "String.length()" because "<parameter1>" is null
	at org.h2.util.ParserUtil.getTokenType(ParserUtil.java:680)
	at org.h2.util.ParserUtil.isKeyword(ParserUtil.java:622)
	at org.h2.command.Parser.isKeyword(Parser.java:5876)
	at org.h2.command.Parser.readIfDataType1(Parser.java:6079)
	at org.h2.command.Parser.readIfDataType(Parser.java:6040)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:6022)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5897)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9328)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:8991)
	at org.h2.command.Parser.parseAlterTable(Parser.java:8461)
	at org.h2.command.Parser.parseAlter(Parser.java:7522)
	at org.h2.command.Parser.parsePrepared(Parser.java:747)
	at org.h2.command.Parser.parse(Parser.java:691)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:568)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	... 7 more
2022-06-09 17:13:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ALTER TABLE CLIENTE[*]"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE CLIENTE [42001-212]
2022-06-09 17:13:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ALTER TABLE CLIENTE [*][USERNAME]"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE CLIENTE [USERNAME] [42001-212]
2022-06-09 17:14:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unknown data type: "USERNAME"; SQL statement:
ALTER TABLE CLIENTE ADD COLOUMN USERNAME [50004-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.readIfDataType1(Parser.java:6199)
	at org.h2.command.Parser.readIfDataType(Parser.java:6040)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:6022)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5897)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9328)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:8991)
	at org.h2.command.Parser.parseAlterTable(Parser.java:8461)
	at org.h2.command.Parser.parseAlter(Parser.java:7522)
	at org.h2.command.Parser.parsePrepared(Parser.java:747)
	at org.h2.command.Parser.parse(Parser.java:691)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:568)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-09 17:14:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.NullPointerException: Cannot invoke ""String.length()"" because ""<parameter1>"" is null" [50000-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.message.DbException.toSQLException(DbException.java:367)
	at org.h2.message.TraceObject.logAndConvert(TraceObject.java:365)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:225)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NullPointerException: Cannot invoke "String.length()" because "<parameter1>" is null
	at org.h2.util.ParserUtil.getTokenType(ParserUtil.java:680)
	at org.h2.util.ParserUtil.isKeyword(ParserUtil.java:622)
	at org.h2.command.Parser.isKeyword(Parser.java:5876)
	at org.h2.command.Parser.readIfDataType1(Parser.java:6079)
	at org.h2.command.Parser.readIfDataType(Parser.java:6040)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:6022)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5897)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9328)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:8991)
	at org.h2.command.Parser.parseAlterTable(Parser.java:8461)
	at org.h2.command.Parser.parseAlter(Parser.java:7522)
	at org.h2.command.Parser.parsePrepared(Parser.java:747)
	at org.h2.command.Parser.parse(Parser.java:691)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:568)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	... 7
2022-06-10 18:10:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "[*]Istruttori (Matricola, nome, cognome, e-mail, telefono)\000d\000a\000d\000aCREATE TABLE ISTRUTTORI(\000d\000aMATRICOLA CHAR(4) NOT NULL PRIMARY KEY,\000d\000aNOME VARCHAR(20),\000d\000aCOGNOME VARCHAR(20),\000d\000aEMAIL VARCHAR(30),\000d\000aTELEFONO VARCHAR(10)"; expected "INSERT"; SQL statement:
Istruttori (Matricola, nome, cognome, e-mail, telefono)

CREATE TABLE ISTRUTTORI(
MATRICOLA CHAR(4) NOT NULL PRIMARY KEY,
NOME VARCHAR(20),
COGNOME VARCHAR(20),
EMAIL VARCHAR(30),
TELEFONO VARCHAR(10) [42001-212]
2022-06-10 18:10:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "[*])"; SQL statement:
) [42000-212]
2022-06-10 18:10:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE ISTRUTTORI(\000d\000aMATRICOLA CHAR(4) NOT NULL PRIMARY KEY,\000d\000aNOME VARCHAR(20),\000d\000aCOGNOME VARCHAR(20),\000d\000aEMAIL VARCHAR(30),\000d\000aTELEFONO VARCHAR(10)[*]"; expected "ARRAY, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, DEFAULT, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, COMMENT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, AUTO_INCREMENT, ,, )"; SQL statement:
CREATE TABLE ISTRUTTORI(
MATRICOLA CHAR(4) NOT NULL PRIMARY KEY,
NOME VARCHAR(20),
COGNOME VARCHAR(20),
EMAIL VARCHAR(30),
TELEFONO VARCHAR(10) [42001-212]
2022-06-10 18:10:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "[*])"; SQL statement:
) [42000-212]
2022-06-10 18:10:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE ISTRUTTORI(\000d\000aMATRICOLA CHAR(4) NOT NULL PRIMARY KEY,\000d\000aNOME VARCHAR(20),\000d\000aCOGNOME VARCHAR(20),\000d\000aEMAIL VARCHAR(30),\000d\000aTELEFONO VARCHAR(10)[*]"; expected "ARRAY, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, DEFAULT, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, COMMENT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, AUTO_INCREMENT, ,, )"; SQL statement:
CREATE TABLE ISTRUTTORI(
MATRICOLA CHAR(4) NOT NULL PRIMARY KEY,
NOME VARCHAR(20),
COGNOME VARCHAR(20),
EMAIL VARCHAR(30),
TELEFONO VARCHAR(10) [42001-212]
2022-06-10 18:10:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "[*])"; SQL statement:
) [42000-212]
2022-06-10 18:27:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Attempt to define a second primary key; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL PRIMARY KEY,
ORA CHAR(5) NOT NULL PRIMARY KEY,
ISTRUTTORE CHAR(4) NOT NULL PRIMARY KEY,
FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)) [90017-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.command.ddl.CommandWithColumns.setPrimaryKey(CommandWithColumns.java:148)
	at org.h2.command.ddl.CommandWithColumns.addConstraintCommand(CommandWithColumns.java:49)
	at org.h2.command.Parser.readColumnConstraints(Parser.java:9381)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9333)
	at org.h2.command.Parser.parseCreateTable(Parser.java:9266)
	at org.h2.command.Parser.parseCreate(Parser.java:6745)
	at org.h2.command.Parser.parsePrepared(Parser.java:763)
	at org.h2.command.Parser.parse(Parser.java:691)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:568)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-10 18:31:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL PRIMARY KEY,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aFOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)\000d\000a[*]CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA))"; expected "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL PRIMARY KEY,
ISTRUTTORE CHAR(4) NOT NULL,
FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)) [42001-212]
2022-06-10 18:32:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL ,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aFOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)\000d\000a[*]CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA))"; expected "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL ,
ISTRUTTORE CHAR(4) NOT NULL,
FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)) [42001-212]
2022-06-10 18:32:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL ,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aFOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)\000d\000a[*]CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA))"; expected "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL ,
ISTRUTTORE CHAR(4) NOT NULL,
FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)) [42001-212]
2022-06-10 18:32:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL ,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aFOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)\000d\000a[*]CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA))"; expected "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL ,
ISTRUTTORE CHAR(4) NOT NULL,
FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)) [42001-212]
2022-06-10 18:33:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL ,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA))"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL ,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)) [42001-212]
2022-06-10 18:39:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL ,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL ,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA)
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)
) [42001-212]
2022-06-10 18:41:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA),\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA),
) [42001-212]
2022-06-10 18:41:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA),\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA),
) [42001-212]
2022-06-10 18:42:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA),\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA),
) [42001-212]
2022-06-10 18:43:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA),\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA),
) [42001-212]
2022-06-10 18:43:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA),\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA),
) [42001-212]
2022-06-10 18:45:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)
) [42001-212]
2022-06-10 18:45:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)
) [42001-212]
2022-06-10 18:45:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)
) [42001-212]
2022-06-10 18:45:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)
) [42001-212]
2022-06-10 18:45:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)
) [42001-212]
2022-06-10 18:45:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)
) [42001-212]
2022-06-10 18:45:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)
) [42001-212]
2022-06-10 18:45:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)
) [42001-212]
2022-06-10 18:46:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)
) [42001-212]
2022-06-10 18:46:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)
) [42001-212]
2022-06-10 18:46:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE DISPONIBILITA(\000d\000aGIORNO VARCHAR(9) NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCONSTRAINT [*]FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aCONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE DISPONIBILITA(
GIORNO VARCHAR(9) NOT NULL,
ORA CHAR(5) NOT NULL,
ISTRUTTORE CHAR(4) NOT NULL,
CONSTRAINT FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
CONSTRAINT pk_disponibilita PRIMARY KEY(ISTRUTTORE, GIORNO, ORA)
) [42001-212]
2022-06-10 19:08:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE [*]LEZIONIGUIDA(\000d\000aIDLEZIONE INT NOT NULL AUTO_INCREMENT PRIMARY KEY,\000d\000aDATA DATE NOT NULL,\000d\000aORA CHAR(5) NOT NULL,\000d\000aDURATA  INT,\000d\000aTIPOPATENTE VARCHAR(4),\000d\000aISTRUTTORE CHAR(4) NOT NULL,\000d\000aCLIENTE CHAR(9) NOT NULL,\000d\000aFOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),\000d\000aFOREIGN KEY (CLIENTE) REFERENCES CLIENTI(CARTAID)\000d\000a)"; expected "OR, FORCE, VIEW, ALIAS, SEQUENCE, USER, TRIGGER, ROLE, SCHEMA, CONSTANT, DOMAIN, TYPE, DATATYPE, AGGREGATE, LINKED, MEMORY, CACHED, LOCAL, GLOBAL, TEMP, TEMPORARY, TABLE, SYNONYM, UNIQUE, HASH, SPATIAL, INDEX"; SQL statement:
CREATE LEZIONIGUIDA(
IDLEZIONE INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
DATA DATE NOT NULL,
ORA CHAR(5) NOT NULL,
DURATA  INT,
TIPOPATENTE VARCHAR(4),
ISTRUTTORE CHAR(4) NOT NULL,
CLIENTE CHAR(9) NOT NULL,
FOREIGN KEY (ISTRUTTORE) REFERENCES ISTRUTTORI(MATRICOLA),
FOREIGN KEY (CLIENTE) REFERENCES CLIENTI(CARTAID)
) [42001-212]
2022-06-10 19:18:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "TIPOPATENTE" not found; SQL statement:
CREATE TABLE ISCRIZIONI(
CLIENTE CHAR(9) NOT NULL,
CORSO VARCHAR(4) NOT NULL,
DATAISCRIZIONE DATE,
DATACONSEGUIMENTO DATE DEFAULT NULL,
CONSTRAINT pk_iscrizioni PRIMARY KEY (CLIENTE, CORSO),
FOREIGN KEY (CLIENTE) REFERENCES CLIENTI(CARTAID),
FOREIGN KEY (CORSO) REFERENCES TIPOPATENTE(CORSO)
) [42102-212]
2022-06-10 19:18:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CORSO" not found; SQL statement:
CREATE TABLE ISCRIZIONI(
CLIENTE CHAR(9) NOT NULL,
CORSO VARCHAR(4) NOT NULL,
DATAISCRIZIONE DATE,
DATACONSEGUIMENTO DATE DEFAULT NULL,
CONSTRAINT pk_iscrizioni PRIMARY KEY (CLIENTE, CORSO),
FOREIGN KEY (CLIENTE) REFERENCES CLIENTI(CARTAID),
FOREIGN KEY (CORSO) REFERENCES CORSO(TIPOPATENTE)
) [42102-212]
2022-06-10 19:32:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "V" not found; SQL statement:
CREATE TABLE DOMANDE(
IDDOMANDA INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
FIGURA VARCHAR(40) DEFAULT NULL,
TEMA VARCHAR(20) NOT NULL,
FORMULAZIONE VARCHAR(100) NOT NULL,
RISPOSTACORRETTA CHAR(1) NOT NULL CHECK(RISPOSTACORRETTA=V OR RISPOSTACORRETTA=F)
) [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:140)
	at org.h2.expression.condition.ConditionAndOr.optimize(ConditionAndOr.java:136)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:191)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.ddl.AlterTable.update(AlterTable.java:46)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:131)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-10 19:32:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "V" not found; SQL statement:
CREATE TABLE DOMANDE(
IDDOMANDA INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
FIGURA VARCHAR(40) DEFAULT NULL,
TEMA VARCHAR(20) NOT NULL,
FORMULAZIONE VARCHAR(100) NOT NULL,
RISPOSTACORRETTA CHAR(1) NOT NULL CHECK(RISPOSTACORRETTA="V" OR RISPOSTACORRETTA="F")
) [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:140)
	at org.h2.expression.condition.ConditionAndOr.optimize(ConditionAndOr.java:136)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:191)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.ddl.AlterTable.update(AlterTable.java:46)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:131)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-10 19:36:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Cannot drop "CORSI" because "CONSTRAINT_5E77" depends on it; SQL statement:
DROP TABLE CORSI [90107-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:97)
	at org.h2.command.ddl.DropTable.update(DropTable.java:121)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-10 19:36:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE CORSI(\000d\000aTIPOPATENTE VARCHAR(4) NOT NULL PRIMARY KEY,\000d\000aCONSTRAINT check_patente CHECK(TIPOPATENTE IN(""A1"", ""A2"", ""AM"", 'A', 'B')\000d\000a)[*]"; expected "., [, ::, AT, FORMAT, *, /, %, +, -, ||, NOT, IS, ILIKE, REGEXP, AND, OR, NOCHECK, CHECK, ,, )"; SQL statement:
CREATE TABLE CORSI(
TIPOPATENTE VARCHAR(4) NOT NULL PRIMARY KEY,
CONSTRAINT check_patente CHECK(TIPOPATENTE IN("A1", "A2", "AM", 'A', 'B')
) [42001-212]
2022-06-10 19:37:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE CORSI(\000d\000aTIPOPATENTE VARCHAR(4) NOT NULL PRIMARY KEY,\000d\000aCONSTRAINT check_patente CHECK(TIPOPATENTE IN('A1', 'A2', 'AM', 'A', 'B')\000d\000a)[*]"; expected "., [, ::, AT, FORMAT, *, /, %, +, -, ||, NOT, IS, ILIKE, REGEXP, AND, OR, NOCHECK, CHECK, ,, )"; SQL statement:
CREATE TABLE CORSI(
TIPOPATENTE VARCHAR(4) NOT NULL PRIMARY KEY,
CONSTRAINT check_patente CHECK(TIPOPATENTE IN('A1', 'A2', 'AM', 'A', 'B')
) [42001-212]
2022-06-10 19:37:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CORSI" already exists; SQL statement:
CREATE TABLE CORSI(
TIPOPATENTE VARCHAR(2) NOT NULL PRIMARY KEY,
CONSTRAINT check_patente CHECK(TIPOPATENTE IN('A1', 'A2', 'AM', 'A', 'B'))
) [42101-212]
2022-06-10 19:37:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CORSI" already exists; SQL statement:
CREATE TABLE CORSI(
TIPOPATENTE VARCHAR(4) NOT NULL PRIMARY KEY,
CONSTRAINT check_patente CHECK(TIPOPATENTE IN('A1', 'A2', 'AM', 'A', 'B'))
) [42101-212]
2022-06-10 19:42:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE PROVE(\000d\000aIDPROVA INT NOT NULL AUTO_INCREMENT PRIMARY KEY,\000d\000aCLIENTE CHAR(9) NOT NULL,\000d\000aDATA DATE NOT NULL,\000d\000aESITO CHAR(1) NOT NULL,\000d\000aCONSTRAINT check_esito CHECK(ESITO IN('p', 'P', 'n', 'N'),\000d\000a[*]FOREIGN KEY (CLIENTE) REFERENCES CLIENTI(CARTAID)\000d\000a)"; expected "INTERSECTS, NOT, EXISTS, UNIQUE, INTERSECTS"; SQL statement:
CREATE TABLE PROVE(
IDPROVA INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
CLIENTE CHAR(9) NOT NULL,
DATA DATE NOT NULL,
ESITO CHAR(1) NOT NULL,
CONSTRAINT check_esito CHECK(ESITO IN('p', 'P', 'n', 'N'),
FOREIGN KEY (CLIENTE) REFERENCES CLIENTI(CARTAID)
) [42001-212]
2022-06-10 19:53:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "FORMULAZIONE CHARACTER VARYING(100)": "U&'Ai passaggi a livello occorre usare la massima prudenza e moderare la velocit... (166)"; SQL statement:
INSERT INTO DOMANDE VALUES(1, NULL, 'Limiti di velocità', 'Ai passaggi a livello occorre usare la massima prudenza e moderare la velocità, per evitare situazioni di pericolo che comportino il danneggiamento delle attrezzature', 'v') [22001-212]
2022-06-10 19:53:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Cannot drop "DOMANDE" because "CONSTRAINT_7C" depends on it; SQL statement:
DROP TABLE DOMANDE [90107-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:97)
	at org.h2.command.ddl.DropTable.update(DropTable.java:121)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-10 19:53:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Cannot drop "PROVE" because "CONSTRAINT_7" depends on it; SQL statement:
DROP TABLE PROVE [90107-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:97)
	at org.h2.command.ddl.DropTable.update(DropTable.java:121)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-10 20:03:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO DOMANDE VALUES(2, NULL, 'Limiti di velocità', 'E' obbligatorio regolare la velocità nelle ore diurne ma non in quelle notturne', 'f[*]')"; SQL statement:
INSERT INTO DOMANDE VALUES(2, NULL, 'Limiti di velocità', 'E' obbligatorio regolare la velocità nelle ore diurne ma non in quelle notturne', 'f') [42000-212]
2022-06-12 00:38:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CLIENTE" not found; SQL statement:
select * from cliente [42102-212]
2022-06-11 23:46:12 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
2022-06-11 23:46:18 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
2022-06-11 23:53:28 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
2022-06-12 01:56:29 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:124)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3349)
	at org.h2.mvstore.MVStore.panic(MVStore.java:590)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:466)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4118)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at Database.DBManager.getConnection(DBManager.java:17)
	at Database.ClienteDAO.selectClienti(ClienteDAO.java:114)
	at Controller.GestioneScuolaGuida.selectClienti(GestioneScuolaGuida.java:78)
	at Boundary.BoundarySegretario.registraCliente(BoundarySegretario.java:29)
	at Boundary.MainMenu.main(MainMenu.java:26)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:449)
	... 16 more
2022-06-12 01:57:03 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:124)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3349)
	at org.h2.mvstore.MVStore.panic(MVStore.java:590)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:466)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4118)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at Database.DBManager.getConnection(DBManager.java:17)
	at Database.ClienteDAO.selectClienti(ClienteDAO.java:114)
	at Controller.GestioneScuolaGuida.selectClienti(GestioneScuolaGuida.java:78)
	at Boundary.BoundarySegretario.registraCliente(BoundarySegretario.java:29)
	at Boundary.MainMenu.main(MainMenu.java:26)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:449)
	... 16 more
2022-06-12 01:57:29 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:124)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3349)
	at org.h2.mvstore.MVStore.panic(MVStore.java:590)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:466)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4118)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at Database.DBManager.getConnection(DBManager.java:17)
	at Database.ClienteDAO.selectClienti(ClienteDAO.java:114)
	at Controller.GestioneScuolaGuida.selectClienti(GestioneScuolaGuida.java:78)
	at Boundary.BoundarySegretario.registraCliente(BoundarySegretario.java:29)
	at Boundary.MainMenu.main(MainMenu.java:26)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:449)
	... 16 more
2022-06-12 02:03:40 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:124)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3349)
	at org.h2.mvstore.MVStore.panic(MVStore.java:590)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:466)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4118)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at Database.DBManager.getConnection(DBManager.java:17)
	at Database.ClienteDAO.selectClienti(ClienteDAO.java:114)
	at Controller.GestioneScuolaGuida.selectClienti(GestioneScuolaGuida.java:78)
	at Boundary.BoundarySegretario.registraCliente(BoundarySegretario.java:29)
	at Boundary.MainMenu.main(MainMenu.java:26)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:449)
	... 16 more
2022-06-12 02:04:50 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:124)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3349)
	at org.h2.mvstore.MVStore.panic(MVStore.java:590)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:466)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4118)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:153)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:449)
	... 16 more
2022-06-12 02:04:58 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:124)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3349)
	at org.h2.mvstore.MVStore.panic(MVStore.java:590)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:466)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4118)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:153)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:449)
	... 16 more
2022-06-12 02:14:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES (? [*]? ? ? ? ? ? ? ? ?);"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, NOT, IS, ILIKE, REGEXP, AND, OR, ,, )"; SQL statement:
INSERT INTO CLIENTI VALUES (? ? ? ? ? ? ? ? ? ?); [42001-212]
2022-06-12 00:16:19 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
2022-06-12 00:27:26 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
2022-06-12 02:47:59 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:124)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3349)
	at org.h2.mvstore.MVStore.panic(MVStore.java:590)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:466)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4118)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at Database.DBManager.getConnection(DBManager.java:17)
	at Database.ClienteDAO.memorizzaCliente(ClienteDAO.java:79)
	at Controller.GestioneScuolaGuida.memorizzaCliente(GestioneScuolaGuida.java:68)
	at Boundary.BoundarySegretario.registraCliente(BoundarySegretario.java:79)
	at Boundary.MainMenu.main(MainMenu.java:26)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/claudio/GitHub/Java/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:449)
	... 16 more
2022-06-12 02:49:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES (? [*]? ? ? ? ? ? ? ? ?);"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, NOT, IS, ILIKE, REGEXP, AND, OR, ,, )"; SQL statement:
INSERT INTO CLIENTI VALUES (? ? ? ? ? ? ? ? ? ?); [42001-212]
2022-06-12 02:51:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES (? [[*]*]? ? ? ? ? ? ? ? ?);"; expected "INTERSECTS, NOT, EXISTS, UNIQUE, INTERSECTS"; SQL statement:
INSERT INTO CLIENTI VALUES (? [*]? ? ? ? ? ? ? ? ?); [42001-212]
2022-06-12 02:55:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES (? [*]UNIQUE ? ? ? ? ? ? ? ? ?);"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, NOT, IS, ILIKE, REGEXP, AND, OR, ,, )"; SQL statement:
INSERT INTO CLIENTI VALUES (? UNIQUE ? ? ? ? ? ? ? ? ?); [42001-212]
2022-06-12 02:58:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI (CARTAID, NOME, COGNOME, DATANASCITA, EMAIL, VIA, NUMEROCIVICO, CAP) VALUES (? [*]UNIQUE ? ? ? ? ? ? ? ? ?);"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, NOT, IS, ILIKE, REGEXP, AND, OR, ,, )"; SQL statement:
INSERT INTO CLIENTI (CARTAID, NOME, COGNOME, DATANASCITA, EMAIL, VIA, NUMEROCIVICO, CAP) VALUES (? UNIQUE ? ? ? ? ? ? ? ? ?); [42001-212]
2022-06-12 03:04:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES (UNIQUE [*]? ? ? ? ? ? ? ? ? ?);"; expected "("; SQL statement:
INSERT INTO CLIENTI VALUES (UNIQUE ? ? ? ? ? ? ? ? ? ?); [42001-212]
2022-06-12 03:08:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES (? [*]? ? ? ? ? ? ?);"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, NOT, IS, ILIKE, REGEXP, AND, OR, ,, )"; SQL statement:
INSERT INTO CLIENTI VALUES (? ? ? ? ? ? ? ?); [42001-212]
2022-06-12 03:10:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES (UNIQUE([*]?) ? ? ? ? ? ? ?);"; expected "WITH, (, SELECT, TABLE, VALUES"; SQL statement:
INSERT INTO CLIENTI VALUES (UNIQUE(?) ? ? ? ? ? ? ?); [42001-212]
2022-06-12 03:16:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', 5, 55555, 'marcello55', '5marcelloPosix5[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', 5, 55555, 'marcello55', '5marcelloPosix5'); [42000-212]
2022-06-12 03:16:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', 5, 55555, 'marcello55', 'marcelloPosix5[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', 5, 55555, 'marcello55', 'marcelloPosix5'); [42000-212]
2022-06-12 03:16:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', 5, '55555', 'marcello55', 'marcelloPosix5[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', 5, '55555', 'marcello55', 'marcelloPosix5'); [42000-212]
2022-06-12 03:17:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix5[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix5'); [42000-212]
2022-06-12 03:17:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix'); [42000-212]
2022-06-12 03:18:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix'); [42000-212]
2022-06-12 03:18:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix'); [42000-212]
2022-06-12 03:18:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix'); [42000-212]
2022-06-12 03:18:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix'); [42000-212]
2022-06-12 03:18:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55'); [42000-212]
2022-06-12 03:19:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555'); [42000-212]
2022-06-12 03:19:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5'); [42000-212]
2022-06-12 03:19:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria'); [42000-212]
2022-06-12 03:19:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it'); [42000-212]
2022-06-12 03:19:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02'); [42000-212]
2022-06-12 03:19:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque'); [42000-212]
2022-06-12 03:19:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello'); [42000-212]
2022-06-12 03:19:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES([*]'MC5555555);"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555); [42000-212]
2022-06-12 03:19:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column count does not match; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555') [21002-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:295)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-12 03:20:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix[*]');"; SQL statement:
INSERT INTO CLIENTI VALUES('MC5555555, 'Marcello', 'Cinque', '1998-09-02', 'cinque@unina.it', 'Lago Patria', '5', '55555', 'marcello55', 'marcelloPosix'); [42000-212]
2022-06-12 03:23:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column count does not match; SQL statement:
INSERT INTO CLIENTI VALUES (?, ?, ?, ?, ?, ?, ?, ?); [21002-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:295)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:92)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:288)
	at Database.ClienteDAO.memorizzaCliente(ClienteDAO.java:82)
	at Controller.GestioneScuolaGuida.memorizzaCliente(GestioneScuolaGuida.java:68)
	at Boundary.BoundarySegretario.registraCliente(BoundarySegretario.java:79)
	at Boundary.MainMenu.main(MainMenu.java:26)
2022-06-12 03:25:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "DATANASCINA" not found; SQL statement:
INSERT INTO CLIENTI (CARTAID, NOME, COGNOME, DATANASCINA, EMAIL, VIA, NUMEROCIVICO, CAP) VALUES (?, ?, ?, ?, ?, ?, ?, ?); [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.table.Table.getColumn(Table.java:754)
	at org.h2.command.Parser.parseColumn(Parser.java:1264)
	at org.h2.command.Parser.parseColumnList(Parser.java:1249)
	at org.h2.command.Parser.parseInsert(Parser.java:1650)
	at org.h2.command.Parser.parsePrepared(Parser.java:814)
	at org.h2.command.Parser.parse(Parser.java:691)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:568)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:92)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:288)
	at Database.ClienteDAO.memorizzaCliente(ClienteDAO.java:82)
	at Controller.GestioneScuolaGuida.memorizzaCliente(GestioneScuolaGuida.java:68)
	at Boundary.BoundarySegretario.registraCliente(BoundarySegretario.java:79)
	at Boundary.MainMenu.main(MainMenu.java:26)
2022-06-12 03:26:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "EMAIL CHARACTER VARYING(20)": "'kurt.cobain@nirvanamail.com' (27)"; SQL statement:
INSERT INTO CLIENTI (CARTAID, NOME, COGNOME, DATANASCITA, EMAIL, VIA, NUMEROCIVICO, CAP) VALUES (?, ?, ?, ?, ?, ?, ?, ?) [22001-212]
2022-06-12 03:28:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "EMAIL CHARACTER VARYING(20)": "'kurt.cobain@nirvana.com' (23)"; SQL statement:
INSERT INTO CLIENTI (CARTAID, NOME, COGNOME, DATANASCITA, EMAIL, VIA, NUMEROCIVICO, CAP) VALUES (?, ?, ?, ?, ?, ?, ?, ?) [22001-212]
2022-06-12 03:41:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "TIPOPATENTE CHARACTER VARYING(4)": "'nessuna' (7)"; SQL statement:
INSERT INTO PATENTIINPOSSESSO VALUES (?, ?, ?) [22001-212]
2022-06-12 03:49:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CLIENTE" not found; SQL statement:
delete from cliente where cartaid = 'AX5059049' [42102-212]
2022-06-12 03:49:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CLIENTE" not found; SQL statement:
delete from cliente where cartaid = 'AX5059049' [42102-212]
2022-06-12 03:49:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CLIENTE" not found; SQL statement:
delete from cliente where cartaid = 'AX5059049' [42102-212]
2022-06-12 03:59:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "select * from clienti [*]and patentiinpossesso"; SQL statement:
select * from clienti and patentiinpossesso [42000-212]
2022-06-12 03:59:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "select * from clienti [*]and patentiinpossesso"; SQL statement:
select * from clienti and patentiinpossesso [42000-212]
2022-06-12 17:29:08 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
2022-06-12 17:48:35 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:359)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:562)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:796)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:677)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:676)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
2022-06-13 10:29:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.autenticazione(ClienteDAO.java:85)
	at Controller.GestioneScuolaGuida.autenticazione(GestioneScuolaGuida.java:68)
	at Boundary.MainMenu.main(MainMenu.java:38)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 6 more
2022-06-13 10:30:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Cinque" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.autenticazione(ClienteDAO.java:85)
	at Controller.GestioneScuolaGuida.autenticazione(GestioneScuolaGuida.java:68)
	at Boundary.MainMenu.main(MainMenu.java:38)
Caused by: java.lang.IllegalArgumentException: Cinque
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 6 more
2022-06-13 10:30:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.autenticazione(ClienteDAO.java:85)
	at Controller.GestioneScuolaGuida.autenticazione(GestioneScuolaGuida.java:68)
	at Boundary.MainMenu.main(MainMenu.java:38)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 6 more
2022-06-13 10:32:59 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/nokia/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:124)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3349)
	at org.h2.mvstore.MVStore.panic(MVStore.java:590)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:466)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4118)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at Database.DBManager.getConnection(DBManager.java:17)
	at Database.ClienteDAO.leggiCliente(ClienteDAO.java:45)
	at Controller.GestioneScuolaGuida.simulaProva(GestioneScuolaGuida.java:114)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:86)
	at Boundary.MainMenu.main(MainMenu.java:41)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/nokia/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/nokia/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:449)
	... 16 more
2022-06-13 10:50:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "ERMVKEVKENVENVE" not found; SQL statement:
select * from clienti where CARTAID = "ERMVKEVKENVENVE" [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:140)
	at org.h2.expression.Expression.optimizeCondition(Expression.java:148)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1177)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-13 11:01:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-13 11:06:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-13 11:07:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-13 11:07:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.leggiCliente(ClienteDAO.java:55)
	at Controller.GestioneScuolaGuida.simulaProva(GestioneScuolaGuida.java:114)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:86)
	at Boundary.MainMenu.main(MainMenu.java:41)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 7 more
2022-06-13 11:07:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.leggiCliente(ClienteDAO.java:55)
	at Controller.GestioneScuolaGuida.simulaProva(GestioneScuolaGuida.java:114)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:86)
	at Boundary.MainMenu.main(MainMenu.java:41)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 7 more
2022-06-13 11:07:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-13 11:09:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.leggiCliente(ClienteDAO.java:55)
	at Controller.GestioneScuolaGuida.simulaProva(GestioneScuolaGuida.java:114)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:86)
	at Boundary.MainMenu.main(MainMenu.java:41)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 7 more
2022-06-13 11:16:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-13 11:16:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-13 11:18:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-13 11:18:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-13 11:19:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-13 11:19:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-13 11:20:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-13 11:20:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-13 11:20:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.leggiCliente(ClienteDAO.java:56)
	at Controller.GestioneScuolaGuida.simulaProva(GestioneScuolaGuida.java:114)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:86)
	at Boundary.MainMenu.main(MainMenu.java:41)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 7 more
2022-06-13 11:25:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.leggiCliente(ClienteDAO.java:59)
	at Controller.GestioneScuolaGuida.simulaProva(GestioneScuolaGuida.java:114)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:86)
	at Boundary.MainMenu.main(MainMenu.java:41)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 7 more
2022-06-13 11:26:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.leggiCliente(ClienteDAO.java:59)
	at Controller.GestioneScuolaGuida.simulaProva(GestioneScuolaGuida.java:114)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:86)
	at Boundary.MainMenu.main(MainMenu.java:41)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 7 more
2022-06-13 14:36:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column count does not match; SQL statement:
INSERT INTO PROVE VALUES (?, ?, ?); [21002-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:295)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:92)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:288)
	at Database.ProvaDAO.createProva(ProvaDAO.java:20)
	at Controller.GestioneScuolaGuida.creaProva(GestioneScuolaGuida.java:211)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:112)
	at Boundary.MainMenu.main(MainMenu.java:41)
2022-06-13 14:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column count does not match; SQL statement:
INSERT INTO PROVE VALUES (?, ?, ?); [21002-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:295)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:92)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:288)
	at Database.ProvaDAO.createProva(ProvaDAO.java:20)
	at Controller.GestioneScuolaGuida.creaProva(GestioneScuolaGuida.java:211)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:112)
	at Boundary.MainMenu.main(MainMenu.java:41)
2022-06-13 14:53:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "4" for parameter "parameterIndex" [90008-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:646)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:298)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1395)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:378)
	at Database.ProvaDAO.createProva(ProvaDAO.java:26)
	at Controller.GestioneScuolaGuida.creaProva(GestioneScuolaGuida.java:211)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:112)
	at Boundary.MainMenu.main(MainMenu.java:41)
2022-06-13 14:58:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column count does not match; SQL statement:
INSERT INTO PROVE(CLIENTE, DATA, ESITO) VALUES (null, ?, ?, ?); [21002-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:295)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:92)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:288)
	at Database.ProvaDAO.createProva(ProvaDAO.java:23)
	at Controller.GestioneScuolaGuida.creaProva(GestioneScuolaGuida.java:211)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:112)
	at Boundary.MainMenu.main(MainMenu.java:41)
2022-06-14 16:16:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SUPERPIPPO2" not found; SQL statement:
update clienti set superpippo2 where CARTAID = 'CA00000AA' [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.table.Table.getColumn(Table.java:754)
	at org.h2.command.Parser.readTableColumn(Parser.java:1092)
	at org.h2.command.Parser.readUpdateSetClause(Parser.java:1159)
	at org.h2.command.Parser.parseUpdate(Parser.java:1128)
	at org.h2.command.Parser.parsePrepared(Parser.java:858)
	at org.h2.command.Parser.parse(Parser.java:691)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:568)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-14 17:20:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.autenticazione(ClienteDAO.java:179)
	at Controller.GestioneScuolaGuida.autenticazione(GestioneScuolaGuida.java:224)
	at Boundary.MainMenu.main(MainMenu.java:43)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 6 more
2022-06-14 17:23:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.autenticazione(ClienteDAO.java:179)
	at Controller.GestioneScuolaGuida.autenticazione(GestioneScuolaGuida.java:224)
	at Boundary.MainMenu.main(MainMenu.java:43)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 6 more
2022-06-14 17:25:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.autenticazione(ClienteDAO.java:179)
	at Controller.GestioneScuolaGuida.autenticazione(GestioneScuolaGuida.java:224)
	at Boundary.MainMenu.main(MainMenu.java:43)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 6 more
2022-06-14 17:28:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROM CLIENTI WHERE USERNAME =[*]= ? AND PASSWORD == ?"; expected "ALL, ANY, SOME"; SQL statement:
SELECT * FROM CLIENTI WHERE USERNAME == ? AND PASSWORD == ? [42001-212]
2022-06-14 17:29:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "select * from clienti where username =[*]= 'superpippo' and password == 'inzaghi'"; expected "ALL, ANY, SOME"; SQL statement:
select * from clienti where username == 'superpippo' and password == 'inzaghi' [42001-212]
2022-06-14 17:31:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.autenticazione(ClienteDAO.java:179)
	at Controller.GestioneScuolaGuida.autenticazione(GestioneScuolaGuida.java:224)
	at Boundary.MainMenu.main(MainMenu.java:43)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 6 more
2022-06-14 17:32:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.autenticazione(ClienteDAO.java:179)
	at Controller.GestioneScuolaGuida.autenticazione(GestioneScuolaGuida.java:224)
	at Boundary.MainMenu.main(MainMenu.java:43)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 6 more
2022-06-14 17:33:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.autenticazione(ClienteDAO.java:179)
	at Controller.GestioneScuolaGuida.autenticazione(GestioneScuolaGuida.java:224)
	at Boundary.MainMenu.main(MainMenu.java:43)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 6 more
2022-06-14 17:35:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.autenticazione(ClienteDAO.java:182)
	at Controller.GestioneScuolaGuida.autenticazione(GestioneScuolaGuida.java:224)
	at Boundary.MainMenu.main(MainMenu.java:43)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 6 more
2022-06-14 17:36:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.autenticazione(ClienteDAO.java:179)
	at Controller.GestioneScuolaGuida.autenticazione(GestioneScuolaGuida.java:224)
	at Boundary.MainMenu.main(MainMenu.java:43)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 6 more
2022-06-14 17:39:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "Pluto" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.ClienteDAO.autenticazione(ClienteDAO.java:179)
	at Controller.GestioneScuolaGuida.autenticazione(GestioneScuolaGuida.java:224)
	at Boundary.MainMenu.main(MainMenu.java:43)
Caused by: java.lang.IllegalArgumentException: Pluto
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 6 more
2022-06-14 17:40:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: No data is available [2000-212]
2022-06-14 17:48:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "fpm@gmail.com" [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.util.LegacyDateTimeUtils.toDate(LegacyDateTimeUtils.java:162)
	at org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:391)
	at Database.IstruttoreDAO.readIstruttore(IstruttoreDAO.java:32)
	at Controller.GestioneScuolaGuida.prenotaLezione(GestioneScuolaGuida.java:38)
	at Boundary.BoundaryCliente.prenotazioneLezione(BoundaryCliente.java:56)
	at Boundary.MainMenu.main(MainMenu.java:59)
Caused by: java.lang.NumberFormatException: For input string: "fpm@gmail"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:668)
	at java.base/java.lang.Integer.parseInt(Integer.java:784)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:190)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 7 more
2022-06-14 18:21:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column count does not match; SQL statement:
INSERT INTO COMPOSIZIONE VALUES (?); [21002-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:295)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:92)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:288)
	at Database.ComposizioneDAO.insertComposizione(ComposizioneDAO.java:19)
	at Controller.GestioneScuolaGuida.insertComposizione(GestioneScuolaGuida.java:225)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:123)
	at Boundary.MainMenu.main(MainMenu.java:61)
2022-06-14 19:06:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Function "PROVE" not found; SQL statement:
SELECT MAX(PROVE(IDPROVA)) FROM PROVE [90022-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.readFunction(Parser.java:3894)
	at org.h2.command.Parser.readTerm(Parser.java:5227)
	at org.h2.command.Parser.readFactor(Parser.java:3440)
	at org.h2.command.Parser.readSum(Parser.java:3427)
	at org.h2.command.Parser.readConcat(Parser.java:3392)
	at org.h2.command.Parser.readCondition(Parser.java:3177)
	at org.h2.command.Parser.readExpression(Parser.java:3093)
	at org.h2.command.Parser.readAggregate(Parser.java:3625)
	at org.h2.command.Parser.readFunction(Parser.java:3874)
	at org.h2.command.Parser.readTerm(Parser.java:5227)
	at org.h2.command.Parser.readFactor(Parser.java:3440)
	at org.h2.command.Parser.readSum(Parser.java:3427)
	at org.h2.command.Parser.readConcat(Parser.java:3392)
	at org.h2.command.Parser.readCondition(Parser.java:3177)
	at org.h2.command.Parser.readExpression(Parser.java:3093)
	at org.h2.command.Parser.parseSelectExpressions(Parser.java:2893)
	at org.h2.command.Parser.parseSelect(Parser.java:2910)
	at org.h2.command.Parser.parseQueryPrimary(Parser.java:2800)
	at org.h2.command.Parser.parseQueryTerm(Parser.java:2667)
	at org.h2.command.Parser.parseQueryExpressionBody(Parser.java:2646)
	at org.h2.command.Parser.parseQueryExpressionBodyAndEndOfQuery(Parser.java:2639)
	at org.h2.command.Parser.parseQueryExpression(Parser.java:2632)
	at org.h2.command.Parser.parseQuery(Parser.java:2593)
	at org.h2.command.Parser.parsePrepared(Parser.java:724)
	at org.h2.command.Parser.parse(Parser.java:691)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:568)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-14 19:06:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT MAX PROVE[*](IDPROVA) FROM PROVE"; SQL statement:
SELECT MAX PROVE(IDPROVA) FROM PROVE [42000-212]
2022-06-14 19:16:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Method is only allowed for a query. Use execute or executeUpdate instead of executeQuery; SQL statement:
INSERT INTO PROVE(CLIENTE, DATA, ESITO) VALUES (?, ?, ?) [90002-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.command.Prepared.query(Prepared.java:235)
	at org.h2.command.CommandContainer.query(CommandContainer.java:256)
	at org.h2.command.Command.executeQuery(Command.java:190)
	at org.h2.jdbc.JdbcPreparedStatement.executeQuery(JdbcPreparedStatement.java:128)
	at Database.ProvaDAO.createProva(ProvaDAO.java:31)
	at Controller.GestioneScuolaGuida.memorizzaProva(GestioneScuolaGuida.java:214)
	at Boundary.BoundaryCliente.simulazioneProva(BoundaryCliente.java:122)
	at Boundary.MainMenu.main(MainMenu.java:61)
2022-06-15 13:42:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO DOMANDE VALUES(1, NULL, 'Limiti di velocità', 'Ai passaggi a livello occorre usare la massima prudenza e moderare la velocità, per evitare situazioni di pericolo che comportino il danneggiamento delle attrezzature', 'v')\000d\000a\000d\000a[*]INSERT INTO DOMANDE VALUES(2, NULL, 'Limiti di velocità', 'E obbligatorio regolare la velocità nelle ore diurne ma non in quelle notturne', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(3, NULL, 'Limiti di velocità', 'L eccessiva velocità in curva può determinare il ribaltamento del veicolo', 'V')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(4, NULL, 'Limiti di velocità', 'I tratti di strada in curva debbono essere percorsi con il pedale della frizione abbassato', 'F')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(5, 'https://domandescuolaguida.it/foto.jpeg', 'Limiti di velocità', 'I contrassegni rappresentati in figura sono applicati sulla parte posteriore di alcune categorie di veicoli', 'V')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(6,NULL, 'Norme veicoli', 'L uso corretto della strada comporta che i veicoli procedano ad una velocità adeguata alle condizioni della strada e del traffico', 'V')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(7,NULL, 'Norme veicoli', 'L uso corretto della strada comporta che si proceda a velocità elevata per non creare intralcio', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(8,NULL, 'Norme veicoli', 'Il carico sugli autoveicoli non deve superare la portata indicata nella carta di circolazione', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(9,NULL, 'Norme veicoli', 'Il carico deve essere segnalato, di giorno, con un panno rosso se sporge dalla parte posteriore', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(10,NULL, 'Norme veicoli', 'Sui motocicli in nessun caso è consentito trasportare un carico', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(11,NULL, 'Distanza sicurezza', 'La distanza di sicurezza è commisurata anche al tipo di veicolo che segue', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(12,NULL, 'Distanza sicurezza', 'La distanza di sicurezza è commisurata anche al limite di velocità riportato sul retro del veicolo che ci precede', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(13,NULL, 'Distanza sicurezza', 'La distanza di sicurezza è una distanza fissa', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(14,NULL, 'Distanza sicurezza', 'La distanza di sicurezza è commisurata anche alle condizioni psicofisiche del conducente', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(15,NULL, 'Distanza sicurezza', 'La distanza di sicurezza va rispettata per non tamponare il veicolo che ci precede', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(16,NULL, 'Fermata e sosta', 'L arresto è l interruzione della marcia del veicolo dovuta ad esigenze della circolazione, come ad esempio ad un semaforo rosso', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(17,NULL, 'Fermata e sosta', 'L arresto è la temporanea sospensione della marcia per esigenze del conducente o dei passeggeri di un veicolo', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(18,NULL, 'Fermata e sosta', 'La fermata per esigenze del conducente può avere la durata massima di dieci minuti', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(19,NULL, 'Fermata e sosta', 'La fermata di un veicolo è vietata se i passeggeri che scendono sono più di uno', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(20,NULL, 'Fermata e sosta', 'La fermata di un veicolo è la temporanea sospensione della marcia per consentire la salita o la discesa delle persone', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(21,'https://domandescuolaguida.it/foto.jpeg2', 'Segnali di obbligo', 'Il segnale raffigurato consente ai conducenti di passare sia a destra che a sinistra di un salvagente', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(22,'https://domandescuolaguida.it/foto.jpeg3', 'Segnali di obbligo', 'Il segnale raffigurato consente il passaggio da ambedue i lati dell ostacolo', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(23,'https://domandescuolaguida.it/foto.jpeg4', 'Segnali di obbligo', 'Il segnale raffigurato è, di norma, preceduto dal segnale ROTATORIA', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(24,'https://domandescuolaguida.it/foto.jpeg5', 'Segnali di obbligo', 'Il segnale raffigurato è un segnale di divieto', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(25,'https://domandescuolaguida.it/foto.jpeg6', 'Segnali di obbligo', 'Il segnale raffigurato indica al conducente gli unici passaggi consentiti', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(26,NULL, 'Norme varie', 'Sulle carreggiate, sulle rampe e sugli svincoli di autostrade e strade extraurbane principali è vietato invertire il senso di marcia e attraversare lo spartitraffico, anche all altezza dei varchi', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(27,NULL, 'Norme varie', 'Sulle carreggiate, sulle rampe e sugli svincoli di autostrade e strade extraurbane principali è vietato percorrere la carreggiata o parte di essa nel senso di marcia opposto a quello consentito', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(28,NULL, 'Norme varie', 'Sulle autostrade e strade extraurbane principali il conducente può utilizzare la corsia di emergenza se il traffico è intenso', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(29,NULL, 'Norme varie', 'Sulle autostrade e strade extraurbane principali il conducente può utilizzare la corsia di emergenza se il traffico è intenso', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(30,NULL, 'Norme varie', 'Sulle corsie per le soste di emergenza delle autostrade è vietato effettuare la retromarcia', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(31,NULL, 'Ambiente', 'Per evitare rumori fastidiosi durante la guida, occorre far controllare che il dispositivo silenziatore (marmitta) sia in buone condizioni di efficienza', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(32,NULL, 'Ambiente', 'L inquinamento acustico aumenta se si usano in modo eccessivo i dispositivi di segnalazione acustica (clacson, trombe)', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(33,NULL, 'Ambiente', 'Per evitare rumori fastidiosi durante la guida non si deve frenare bruscamente, se non è strettamente necessario', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(34,NULL, 'Ambiente', 'Per evitare rumori fastidiosi non bisogna accelerare inutilmente e ripetutamente se il veicolo è fermo', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(35,NULL, 'Ambiente', 'Per evitare rumori fastidiosi durante la guida, si deve usare il clacson solamente per motivi di sicurezza stradale (trasporto feriti o ammalati, pericolo immediato)', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(36,NULL, 'Ambiente', 'Per evitare rumori fastidiosi durante le fermate dovute al traffico, occorre mantenere il motore acceso, accelerando ripetutamente', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(37,NULL, 'Ambiente', 'Per evitare rumori fastidiosi, occorre chiudere con forza le portiere dell autovettura, in modo da non doverle richiudere', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(38,NULL, 'Ambiente', 'Per evitare rumori fastidiosi durante la guida, occorre sostituire il clacson con le luci ad intermittenza, ma solamente di notte', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(39,NULL, 'Ambiente', 'Per diminuire l inquinamento atmosferico provocato dai veicoli, bisogna far sostituire periodicamente il filtro dell aria', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(40,NULL, 'Ambiente', 'Per diminuire l inquinamento dell aria non bisogna accelerare e frenare spesso durante la marcia del veicolo', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(41,'https.//foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo mostrato in figura A indica i metri di strada già fatta', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(42,'https.//foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo raffigurato indica la pendenza di una strada', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(43,'https.//foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo mostrato in figura A indica la distanza di sicurezza da mantenere dal veicolo che sta davanti', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(44,'https.//foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo raffigurato indica la larghezza di una strada', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(45,'https.//foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo raffigurato, posto sotto il segnale di pericolo, indica la lunghezza del tratto di strada pericoloso', 'f')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(46,'https.//foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo raffigurato, posto sotto un segnale di pericolo, indica la lunghezza del tratto di strada pericoloso', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(47,'https://foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo raffigurato può essere posto sotto un segnale di divieto', 'v')\000d\000a\000d\000aINSERT INTO DOMANDE VALUES(48,'https://foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo raffigurato può essere posto sotto un segnale di divieto', 'v')"; SQL statement:
INSERT INTO DOMANDE VALUES(1, NULL, 'Limiti di velocità', 'Ai passaggi a livello occorre usare la massima prudenza e moderare la velocità, per evitare situazioni di pericolo che comportino il danneggiamento delle attrezzature', 'v')

INSERT INTO DOMANDE VALUES(2, NULL, 'Limiti di velocità', 'E obbligatorio regolare la velocità nelle ore diurne ma non in quelle notturne', 'f')

INSERT INTO DOMANDE VALUES(3, NULL, 'Limiti di velocità', 'L eccessiva velocità in curva può determinare il ribaltamento del veicolo', 'V')

INSERT INTO DOMANDE VALUES(4, NULL, 'Limiti di velocità', 'I tratti di strada in curva debbono essere percorsi con il pedale della frizione abbassato', 'F')

INSERT INTO DOMANDE VALUES(5, 'https://domandescuolaguida.it/foto.jpeg', 'Limiti di velocità', 'I contrassegni rappresentati in figura sono applicati sulla parte posteriore di alcune categorie di veicoli', 'V')

INSERT INTO DOMANDE VALUES(6,NULL, 'Norme veicoli', 'L uso corretto della strada comporta che i veicoli procedano ad una velocità adeguata alle condizioni della strada e del traffico', 'V')

INSERT INTO DOMANDE VALUES(7,NULL, 'Norme veicoli', 'L uso corretto della strada comporta che si proceda a velocità elevata per non creare intralcio', 'f')

INSERT INTO DOMANDE VALUES(8,NULL, 'Norme veicoli', 'Il carico sugli autoveicoli non deve superare la portata indicata nella carta di circolazione', 'v')

INSERT INTO DOMANDE VALUES(9,NULL, 'Norme veicoli', 'Il carico deve essere segnalato, di giorno, con un panno rosso se sporge dalla parte posteriore', 'f')

INSERT INTO DOMANDE VALUES(10,NULL, 'Norme veicoli', 'Sui motocicli in nessun caso è consentito trasportare un carico', 'f')

INSERT INTO DOMANDE VALUES(11,NULL, 'Distanza sicurezza', 'La distanza di sicurezza è commisurata anche al tipo di veicolo che segue', 'f')

INSERT INTO DOMANDE VALUES(12,NULL, 'Distanza sicurezza', 'La distanza di sicurezza è commisurata anche al limite di velocità riportato sul retro del veicolo che ci precede', 'f')

INSERT INTO DOMANDE VALUES(13,NULL, 'Distanza sicurezza', 'La distanza di sicurezza è una distanza fissa', 'f')

INSERT INTO DOMANDE VALUES(14,NULL, 'Distanza sicurezza', 'La distanza di sicurezza è commisurata anche alle condizioni psicofisiche del conducente', 'v')

INSERT INTO DOMANDE VALUES(15,NULL, 'Distanza sicurezza', 'La distanza di sicurezza va rispettata per non tamponare il veicolo che ci precede', 'v')

INSERT INTO DOMANDE VALUES(16,NULL, 'Fermata e sosta', 'L arresto è l interruzione della marcia del veicolo dovuta ad esigenze della circolazione, come ad esempio ad un semaforo rosso', 'v')

INSERT INTO DOMANDE VALUES(17,NULL, 'Fermata e sosta', 'L arresto è la temporanea sospensione della marcia per esigenze del conducente o dei passeggeri di un veicolo', 'f')

INSERT INTO DOMANDE VALUES(18,NULL, 'Fermata e sosta', 'La fermata per esigenze del conducente può avere la durata massima di dieci minuti', 'f')

INSERT INTO DOMANDE VALUES(19,NULL, 'Fermata e sosta', 'La fermata di un veicolo è vietata se i passeggeri che scendono sono più di uno', 'f')

INSERT INTO DOMANDE VALUES(20,NULL, 'Fermata e sosta', 'La fermata di un veicolo è la temporanea sospensione della marcia per consentire la salita o la discesa delle persone', 'v')

INSERT INTO DOMANDE VALUES(21,'https://domandescuolaguida.it/foto.jpeg2', 'Segnali di obbligo', 'Il segnale raffigurato consente ai conducenti di passare sia a destra che a sinistra di un salvagente', 'v')

INSERT INTO DOMANDE VALUES(22,'https://domandescuolaguida.it/foto.jpeg3', 'Segnali di obbligo', 'Il segnale raffigurato consente il passaggio da ambedue i lati dell ostacolo', 'v')

INSERT INTO DOMANDE VALUES(23,'https://domandescuolaguida.it/foto.jpeg4', 'Segnali di obbligo', 'Il segnale raffigurato è, di norma, preceduto dal segnale ROTATORIA', 'v')

INSERT INTO DOMANDE VALUES(24,'https://domandescuolaguida.it/foto.jpeg5', 'Segnali di obbligo', 'Il segnale raffigurato è un segnale di divieto', 'v')

INSERT INTO DOMANDE VALUES(25,'https://domandescuolaguida.it/foto.jpeg6', 'Segnali di obbligo', 'Il segnale raffigurato indica al conducente gli unici passaggi consentiti', 'v')

INSERT INTO DOMANDE VALUES(26,NULL, 'Norme varie', 'Sulle carreggiate, sulle rampe e sugli svincoli di autostrade e strade extraurbane principali è vietato invertire il senso di marcia e attraversare lo spartitraffico, anche all altezza dei varchi', 'v')

INSERT INTO DOMANDE VALUES(27,NULL, 'Norme varie', 'Sulle carreggiate, sulle rampe e sugli svincoli di autostrade e strade extraurbane principali è vietato percorrere la carreggiata o parte di essa nel senso di marcia opposto a quello consentito', 'v')

INSERT INTO DOMANDE VALUES(28,NULL, 'Norme varie', 'Sulle autostrade e strade extraurbane principali il conducente può utilizzare la corsia di emergenza se il traffico è intenso', 'v')

INSERT INTO DOMANDE VALUES(29,NULL, 'Norme varie', 'Sulle autostrade e strade extraurbane principali il conducente può utilizzare la corsia di emergenza se il traffico è intenso', 'f')

INSERT INTO DOMANDE VALUES(30,NULL, 'Norme varie', 'Sulle corsie per le soste di emergenza delle autostrade è vietato effettuare la retromarcia', 'v')

INSERT INTO DOMANDE VALUES(31,NULL, 'Ambiente', 'Per evitare rumori fastidiosi durante la guida, occorre far controllare che il dispositivo silenziatore (marmitta) sia in buone condizioni di efficienza', 'v')

INSERT INTO DOMANDE VALUES(32,NULL, 'Ambiente', 'L inquinamento acustico aumenta se si usano in modo eccessivo i dispositivi di segnalazione acustica (clacson, trombe)', 'v')

INSERT INTO DOMANDE VALUES(33,NULL, 'Ambiente', 'Per evitare rumori fastidiosi durante la guida non si deve frenare bruscamente, se non è strettamente necessario', 'v')

INSERT INTO DOMANDE VALUES(34,NULL, 'Ambiente', 'Per evitare rumori fastidiosi non bisogna accelerare inutilmente e ripetutamente se il veicolo è fermo', 'v')

INSERT INTO DOMANDE VALUES(35,NULL, 'Ambiente', 'Per evitare rumori fastidiosi durante la guida, si deve usare il clacson solamente per motivi di sicurezza stradale (trasporto feriti o ammalati, pericolo immediato)', 'v')

INSERT INTO DOMANDE VALUES(36,NULL, 'Ambiente', 'Per evitare rumori fastidiosi durante le fermate dovute al traffico, occorre mantenere il motore acceso, accelerando ripetutamente', 'f')

INSERT INTO DOMANDE VALUES(37,NULL, 'Ambiente', 'Per evitare rumori fastidiosi, occorre chiudere con forza le portiere dell autovettura, in modo da non doverle richiudere', 'f')

INSERT INTO DOMANDE VALUES(38,NULL, 'Ambiente', 'Per evitare rumori fastidiosi durante la guida, occorre sostituire il clacson con le luci ad intermittenza, ma solamente di notte', 'f')

INSERT INTO DOMANDE VALUES(39,NULL, 'Ambiente', 'Per diminuire l inquinamento atmosferico provocato dai veicoli, bisogna far sostituire periodicamente il filtro dell aria', 'v')

INSERT INTO DOMANDE VALUES(40,NULL, 'Ambiente', 'Per diminuire l inquinamento dell aria non bisogna accelerare e frenare spesso durante la marcia del veicolo', 'v')

INSERT INTO DOMANDE VALUES(41,'https.//foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo mostrato in figura A indica i metri di strada già fatta', 'f')

INSERT INTO DOMANDE VALUES(42,'https.//foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo raffigurato indica la pendenza di una strada', 'f')

INSERT INTO DOMANDE VALUES(43,'https.//foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo mostrato in figura A indica la distanza di sicurezza da mantenere dal veicolo che sta davanti', 'f')

INSERT INTO DOMANDE VALUES(44,'https.//foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo raffigurato indica la larghezza di una strada', 'f')

INSERT INTO DOMANDE VALUES(45,'https.//foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo raffigurato, posto sotto il segnale di pericolo, indica la lunghezza del tratto di strada pericoloso', 'f')

INSERT INTO DOMANDE VALUES(46,'https.//foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo raffigurato, posto sotto un segnale di pericolo, indica la lunghezza del tratto di strada pericoloso', 'v')

INSERT INTO DOMANDE VALUES(47,'https://foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo raffigurato può essere posto sotto un segnale di divieto', 'v')

INSERT INTO DOMANDE VALUES(48,'https://foto.jpeg', 'Pannelli integrativi', 'Il pannello integrativo raffigurato può essere posto sotto un segnale di divieto', 'v') [42000-212]
2022-06-15 13:43:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO DOMANDE VALUES(16,NULL, 'Fermata e sosta', 'L arresto è l interruzione della marcia del veicolo dovuta ad esigenze della circolazione, come ad esempio ad un semaforo rosso', 'v')\000d\000a\000d\000a[*]INSERT INTO DOMANDE VALUES(17,NULL, 'Fermata e sosta', 'L arresto è la temporanea sospensione della marcia per esigenze del conducente o dei passeggeri di un veicolo', 'f')"; SQL statement:
INSERT INTO DOMANDE VALUES(16,NULL, 'Fermata e sosta', 'L arresto è l interruzione della marcia del veicolo dovuta ad esigenze della circolazione, come ad esempio ad un semaforo rosso', 'v')

INSERT INTO DOMANDE VALUES(17,NULL, 'Fermata e sosta', 'L arresto è la temporanea sospensione della marcia per esigenze del conducente o dei passeggeri di un veicolo', 'f') [42000-212]
2022-06-17 10:51:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "COLOUMN" not found; SQL statement:
alter table clienti
alter coloumn username varchar(20) unique [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.table.Table.getColumn(Table.java:770)
	at org.h2.command.Parser.columnIfTableExists(Parser.java:8907)
	at org.h2.command.Parser.parseAlterTableAlter(Parser.java:8483)
	at org.h2.command.Parser.parseAlterTable(Parser.java:8469)
	at org.h2.command.Parser.parseAlter(Parser.java:7522)
	at org.h2.command.Parser.parsePrepared(Parser.java:747)
	at org.h2.command.Parser.parse(Parser.java:691)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:568)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-17 10:51:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "alter table clienti\000d\000aalter column username varchar(20) [*]unique"; SQL statement:
alter table clienti
alter column username varchar(20) unique [42000-212]
2022-06-17 10:51:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "alter table clienti\000d\000aalter column username varchar(20) [*]unique"; SQL statement:
alter table clienti
alter column username varchar(20) unique [42000-212]
2022-06-17 10:52:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "alter table clienti\000d\000a[*]modify column username varchar(20) unique"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
alter table clienti
modify column username varchar(20) unique [42001-212]
2022-06-17 10:54:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "alter table clienti\000d\000aalter column username set [*]unique"; SQL statement:
alter table clienti
alter column username set unique [42000-212]
2022-06-17 10:54:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "alter table clienti\000d\000aalter column username set [*]unique"; SQL statement:
alter table clienti
alter column username set unique [42000-212]
2022-06-17 10:56:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.NullPointerException: Cannot invoke ""String.length()"" because ""<parameter1>"" is null" [50000-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.message.DbException.toSQLException(DbException.java:367)
	at org.h2.message.TraceObject.logAndConvert(TraceObject.java:365)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:225)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NullPointerException: Cannot invoke "String.length()" because "<parameter1>" is null
	at org.h2.util.ParserUtil.getTokenType(ParserUtil.java:680)
	at org.h2.util.ParserUtil.isKeyword(ParserUtil.java:622)
	at org.h2.command.Parser.isKeyword(Parser.java:5876)
	at org.h2.command.Parser.readIfDataType1(Parser.java:6079)
	at org.h2.command.Parser.readIfDataType(Parser.java:6040)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:6022)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5897)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9328)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:8991)
	at org.h2.command.Parser.parseAlterTable(Parser.java:8461)
	at org.h2.command.Parser.parseAlter(Parser.java:7522)
	at org.h2.command.Parser.parsePrepared(Parser.java:747)
	at org.h2.command.Parser.parse(Parser.java:691)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:568)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	... 7 more
2022-06-17 10:57:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Cannot drop "CLIENTI" because "CONSTRAINT_40F, CONSTRAINT_5E7, CONSTRAINT_48D" depends on it; SQL statement:
drop table clienti [90107-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:97)
	at org.h2.command.ddl.DropTable.update(DropTable.java:121)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-06-17 10:58:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "alter table clienti \000d\000aalter column username set [*]unique"; SQL statement:
alter table clienti 
alter column username set unique [42000-212]
2022-06-17 10:59:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "alter table clienti \000d\000aalter column USERNAME set [*]UNIQUE"; SQL statement:
alter table clienti 
alter column USERNAME set UNIQUE [42000-212]
2022-06-17 22:33:27 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/nokia/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:124)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3349)
	at org.h2.mvstore.MVStore.panic(MVStore.java:590)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:466)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4118)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at Database.DBManager.getConnection(DBManager.java:17)
	at Database.ClienteDAO.readCliente(ClienteDAO.java:19)
	at Controller.GestioneScuolaGuida.readCliente(GestioneScuolaGuida.java:44)
	at Boundary.MainMenu.main(MainMenu.java:42)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/nokia/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]" [50000-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 21 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/nokia/canale_j_z-nirvana/ScuolaGuidaNirvana/lib/database.mv.db [2.1.212/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:449)
	... 15 more
2022-06-17 22:52:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "insert into disponibilita ([*]'MONDAY', '17:00', 'C001')"; expected "identifier"; SQL statement:
insert into disponibilita ('MONDAY', '17:00', 'C001') [42001-212]
2022-06-17 23:26:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "insert into iscrizioni values('CA00000AA', 'B', 2022-06-06[*]', null)"; SQL statement:
insert into iscrizioni values('CA00000AA', 'B', 2022-06-06', null) [42000-212]
2022-06-17 23:26:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "insert into iscrizioni values('CA00000AA', 'B', 2022-06-06', 'null[*]')"; SQL statement:
insert into iscrizioni values('CA00000AA', 'B', 2022-06-06', 'null') [42000-212]
2022-06-17 23:26:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "insert into iscrizioni values('CA00000AA', 'B', 2022-06-06', 'null[*]');"; SQL statement:
insert into iscrizioni values('CA00000AA', 'B', 2022-06-06', 'null'); [42000-212]
2022-06-17 23:27:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "null"; SQL statement:
insert into iscrizioni values('CA00000AA', 'B', '2022-06-06', 'null') [22007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.value.ValueDate.parse(ValueDate.java:53)
	at org.h2.value.Value.convertToDate(Value.java:1854)
	at org.h2.value.Value.convertTo(Value.java:1147)
	at org.h2.value.Value.convertForAssignTo(Value.java:1092)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:369)
	at org.h2.table.Table.convertInsertRow(Table.java:931)
	at org.h2.command.dml.Insert.insertRows(Insert.java:167)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalArgumentException: null
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:187)
	at org.h2.value.ValueDate.parse(ValueDate.java:51)
	... 19 more
2022-06-17 23:28:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "insert into patentiinpossesso([*]'A', 'CA00000AA', '1990-12-12')"; expected "identifier"; SQL statement:
insert into patentiinpossesso('A', 'CA00000AA', '1990-12-12') [42001-212]
2022-06-17 23:28:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "insert into patentiinpossesso([*]'A', 'CA00000AA', '1990-12-12')"; expected "identifier"; SQL statement:
insert into patentiinpossesso('A', 'CA00000AA', '1990-12-12') [42001-212]
2022-06-17 23:30:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PATENTINPOSSESSO" not found; SQL statement:
insert into patentinpossesso values('A', 'CA00000AA', '2005-01-01') [42102-212]
